legate/core/operation.py

```diff
@@ -1463,21 +1470,26 @@ def launch(self, strategy: Strategy) -> None:
             for proj_fn in proj_fns:
                 launcher.add_input(input, ipart.get_requirement(1, proj_fn))
 
-            output = self._context.create_store(input.type)
+            num_tasks = (fan_in + self._radix - 1) // self._radix
+            fan_in = num_tasks
+            done = fan_in == 1
+
+            if done:
+                output = self._outputs[0]
+            else:
+                output = self._context.create_store(input.type)
```

Comment:
defining `done` seems superfluous since it is not used anywhere else. Suggest just doing 
```
    if fan_in == 1:
```
directly

Reply:
[473a832](https://github.com/nv-legate/legate.core/pull/699/commits/473a832f369229b882464d3e232bd40fa579eec2)